docs/source/deepinv.loss.rst

```diff
@@ -93,19 +93,58 @@ Training is implemented using :class:`deepinv.training.AdversarialTrainer` which
 
 Metrics
 --------
-Metrics are generally used to evaluate the performance of a model. Some of them can be used as training losses as well.
+Metrics are generally used to evaluate the performance of a model.
+
+Metrics inherit from the :class:`deepinv.loss.metric.Metric` baseclass, and take either ``x_net, x``
```

Comment:
deepinv.loss.Metric

Reply:
See above that you can import either `deepinv.metric.Metric`, `deepinv.loss.Metric`or `deepinv.loss.metric.Metric`. I've added a line explaining this. However annoyingly it doesn't create the link if I shorten it, so it will have to be verbose

Reply:
renaming them also emphasises that these are metrics, not normal losses, i.e. you can't use physics and model with them